kernel/enter.c

```diff
@@ -585,6 +575,12 @@ int myst_enter_kernel(myst_kernel_args_t* args)
     __options.have_syscall_instruction = args->have_syscall_instruction;
     __options.export_ramfs = args->export_ramfs;
 
+#if !defined(MYST_RELEASE)
+    /* Disable debug malloc if not Linux (which sets image_data to null) */
```

Comment:
We only enable debug malloc for Linux? Should be "Enable debug malloc ..."?

Reply:
Good observation. We don't because image_data is null on Linux, not allowing us to get backtraces.

Reply:
Fixed